### PR TITLE
Minor Grammer fixes to docs

### PR DESCRIPTION
**Pull request by jeffin143**
Grammar fixes

- [x] Read and sign the [CLA][1]. PRs that haven't signed it won't be accepted.
- [x] Make sure your PR complies with the [writer's guide][2].
- [x] Review the diff carefully as sometimes this can reveal issues.
- [x] Do not abandon your Pull Request: [Stale Pull Requests][3].

### CODE REVIEW

File: src/content/plugins/commons-chunk-plugin.md [hunk 1]
```diff
@@ -177,7 +177,7 @@ new webpack.optimize.CommonsChunkPlugin({
 
 ### Passing the `minChunks` property a function
 
-You also have the ability to pass the `minChunks` property a function. This function is called by the `CommonsChunkPlugin` and calls the function with `module` and `count` arguments.
+You also can pass the `minChunks` property a function. This function is called by the `CommonsChunkPlugin` and calls the function with `module` and `count` arguments.
```
Comment (jamesgeorge007) [inline src/content/plugins/commons-chunk-plugin.md]:
```suggestion
You can also pass the `minChunks` property a function. This function is called by the `CommonsChunkPlugin` and calls the function with `module` and `count` arguments.
```

Comment (wizardofhogwarts) [inline src/content/plugins/commons-chunk-plugin.md]: Current version was fine

Comment (EugeneHlushko) [reply src/content/plugins/commons-chunk-plugin.md:180]:
```suggestion
You also have the ability to pass the `minChunks` property a function. This function is called by the `CommonsChunkPlugin` and calls the function with `module` and `count` arguments.
```

**Comment by EugeneHlushko** [comment]
Thanks!

**Pull request merged**